cli.ts

```diff
@@ -309,6 +310,61 @@ const postInitCLI = (cliConfig: CLIConfig, env: EnvVars, args: DenoArgs, parsedA
 				default: false,
 				boolean: true,
 			})
+			.command(
+				'add-contract <sourceFile>',
+				i18n.__('addContractDesc'),
+				(yargs: Arguments) => {
+					yargs.positional('sourceFile', {
+						describe: i18n.__('addSourceFileDesc'),
+						type: 'string',
+						required: true,
+					});
+
+					yargs.option('contractName', {
+						alias: ['name', 'n'],
+						type: 'string',
+					});
+				},
+				(inputArgs: Record<string, unknown>) =>
+					pipe(
+						SanitizedArgs.ofAddContractArgs(inputArgs),
+						chain(args => addContract(args, i18n)),
+						map(renderTable),
+						forkCatch(displayError(cliConfig))(displayError(cliConfig))(identity),
+					),
+			)
+			.command(
+				'rm-contract <contractName>',
+				i18n.__('removeContractDesc'),
+				(yargs: Arguments) => {
+					yargs.positional('contractName', {
+						describe: i18n.__('removeContractNameDesc'),
+						type: 'string',
+						required: true,
+					});
+				},
+				(inputArgs: Record<string, unknown>) =>
+					pipe(
+						SanitizedArgs.ofRemoveContractsArgs(inputArgs),
+						chain(args => removeContract(args, i18n)),
+						map(renderTable),
+						forkCatch(displayError(cliConfig))(displayError(cliConfig))(identity),
+					),
+			)
+			.alias('remove-contract', 'rm-contract')
+			.command(
+				'list-contracts',
+				i18n.__('listContractsDesc'),
+				() => {},
+				(inputArgs: Record<string, unknown>) =>
+					pipe(
+						SanitizedArgs.of(inputArgs),
+						chain(args => listContracts(args, i18n)),
+						map(renderTable),
+						forkCatch(displayError(cliConfig))(displayError(cliConfig))(identity),
```

Comment:
Do we have to do `forkCatch` here (instead of doing it at the end of the `run` function)? Is this a `yargs` limitation again?

Reply:
Buit-in functions use forkCatch as they don't require plugins to be loaded.